sim/hunter/pet.go

```diff
@@ -188,7 +188,8 @@ func (hp *HunterPet) Initialize() {
 	hp.focusDump = hp.NewPetAbility(hp.config.FocusDump, false)
 
 	focusRegenMultiplier := (1.0 + 0.1*float64(hp.hunterOwner.Talents.BestialDiscipline)) *
-		core.TernaryFloat64(hp.hunterOwner.HasRune(proto.HunterRune_RuneHandsBeastmastery), 1.5, 1.0)
+		core.TernaryFloat64(hp.hunterOwner.HasRune(proto.HunterRune_RuneHandsBeastmastery), 1.5, 1.0) *
+		core.TernaryFloat64(hp.hunterOwner.HasPredatorArmor[5], 1.2, 1.0)
```

Comment:
To allow this to be done inside of the item set code, we could expose a method for modifying the focus regen multiplier in `focus.go`. I took a similar approach in the rage code in order to apply some of warrior's dynamic rage multipliers

Reply:
That would also allow us to apply the Beast Mastery multiplier inside that rune's code